projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -189,7 +189,7 @@ const Article = ({
     })
     const isConnected = data && data.netInfo.isConnected
     const shareUrlFetchEnabled =
-        !shareUrl &&
+        !article.webUrl &&
```

Comment:
if I set this to shareUrl (mutable state), then when shareUrl is updated by `updateShareUrl` then shareUrlFetchEnabled will update as well. This solves that.